docs/basics/userguide/save_load.md

```diff
@@ -0,0 +1,158 @@
+    agent_name="PPO" + env_id,  # Name of the agent
+    output_dir="./results/",  # where to store the outpus
+)
+
+experiment_to_save.fit()
+print(experiment_to_save.get_agent_instances()[0].Q)  # print the content of the Q-table
+```
+Then save:
+```python
+experiment_to_save.save()  # save the model inside the 'output_dir'
+```
+
+```none
+[INFO] 11:11: Running ExperimentManager fit() for PPOFrozenLake-v1 with n_fit = 1 and max_workers = None.
+[INFO] 11:11:                                    agent_name     worker  episode_rewards  max_global_step
+                                              PPOFrozenLake-v1    0          0.0             178711
+[INFO] 11:11: ... trained!
+/home/jteigny/my_projects/work_in_progress/rlberry/rlberry_userguide/user_guide/rlberry/utils/writers.py:108: FutureWarning: The behavior of DataFrame concatenation with empty or all-NA entries is deprecated. In a future version, this will no longer exclude empty or all-NA columns when determining the result dtypes. To retain the old behavior, exclude the relevant entries before the concat operation.
+  df = pd.concat([df, pd.DataFrame(self._data[tag])], ignore_index=True)
+[[0.73509189 0.77378094 0.77378094 0.73509189]
+ [0.73509189 0.         0.81450625 0.77378094]
+ [0.77378094 0.857375   0.77378094 0.81450625]
+ [0.81450625 0.         0.77377103 0.77378092]
+ [0.77378094 0.81450625 0.         0.73509189]
+ [0.         0.         0.         0.        ]
+ [0.         0.9025     0.         0.81450625]
+ [0.         0.         0.         0.        ]
+ [0.81450625 0.         0.857375   0.77378094]
+ [0.81450625 0.9025     0.9025     0.        ]
+ [0.857375   0.95       0.         0.857375  ]
+ [0.         0.         0.         0.        ]
+ [0.         0.         0.         0.        ]
+ [0.         0.9025     0.95       0.857375  ]
+ [0.9025     0.95       1.         0.9025    ]
+ [0.         0.         0.         0.        ]]
+[INFO] 11:11: Saved ExperimentManager(PPOFrozenLake-v1) using pickle.
+```
+
+After this run, you can see the 'print' of the q-table, and should find a folder named according to the `output_dir` parameter (here `results`), with the data about your experiment.
+In this folder, you should find :
+- `manager_obj.pickle` and folder `agent_handler`, the save of your experiment and your agent.
+- `data.csv`, the episodes result during the training process
+
+
+## How to load a previous experiment?
+In this example you will load the experiment saved in the part 1.
+
+To load an experiment previously saved, you need to :
+- Locate the file you want to load
+- use the function `load()` from the class [ExperimentManager](rlberry.manager.ExperimentManager.load).
+
+```python
+import pathlib
+from rlberry.envs import gym_make
+from rlberry.manager.experiment_manager import ExperimentManager
+
+
+path_to_load = next(
+    pathlib.Path("results").glob("**/manager_obj.pickle")
+)  # find the path to the "manager_obj.pickle"
```

Comment:
Remark: this could be included into rlberry in fact, have a parameter in the load like load from this and you would have to give the directory name (results here).

Reply:
TODO : https://github.com/rlberry-py/rlberry/issues/407